src/quest_manager/templates/quest_manager/quest_approval.html

```diff
@@ -72,6 +72,54 @@
 
     $('[data-toggle="tooltip"]').tooltip();
 });
+
+// all forms that has an id starting with "quick_reply"
+$("form[id^='quick_reply']").submit(function (event) {
+
+  // get form data to send
+  var formData = new FormData(this);
+  // so approve view knows which button was pressed
+  var submit_name = event.originalEvent.submitter.name;
+  formData.append(submit_name, '');
+
+  // if not any of these buttons return.
+  // (comment_button, add_text_button)
+  if (!['approve_button', 'return_button'].includes(submit_name)) {
+    return;
+  }
+
+  // stops the form from refreshing page (but also sending data)
```

Comment:
Why?  Code comments explaining "why" are more valuable than explaining "What' (though "what" is also helpful!)

Reply:
adding a doc string will make the comment clearer

```
/*
  Overrides basic form function (refresh page and redirect to new url)
  To send ajax request, remove submission container, and update messages dynamically
  */
```